_pages/openmic.markdown

```diff
@@ -0,0 +1,29 @@
+---
+title: "Open mic sessions"
+layout: single
+permalink: open-mic
+taxonomy: OpenMic
+
+lang: en
+lang-ref: open-mic
+#classes: wide
+author_profile: true
+
+header:
+  image: /assets/images/archive/openmic.jpg
+  caption: "Photo credit: [**Matthias Wagner**](https://unsplash.com/@matwag?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText) on [**Unsplash**](http://unsplash.com/)"
+---
+
+Open mic session is an activity going on irregulary every two weeks. A member of a team or a guest presents an interesting topic to the rest of the team and anyone willig to listen.
+
+Upcoming event:
+
+{% for post in site.categories.["Open Mic Announcement"] limit:1 %}
```

Comment:
I am seeing some Liquid syntax warnings for the `site.categories.["Open Mic Announcement"]`. Specifically the part in the square brackets. Is there a better way of specifying the category?

Reply:
Yes, I was looking into that too, that is caused by the multiword category name. Two possible solutions - have only one word categories, or have them divided by minus, but that will be visible everywhere. I was looking if there is something like label and id for categories, but there is not.

Reply:
ok, too bad.